packages/deployments/contracts/test/connext.spec.ts

```diff
@@ -132,11 +137,30 @@ describe("Connext", () => {
     const connextUtils = await deployContract("ConnextUtils");
     const routerPermissionsManagerLogic = await deployContract("RouterPermissionsManagerLogic");
 
+    // Deploy RelayerFeeRouters
+    originRelayerFeeRouter = await deployUpgradeableProxy<RelayerFeeRouter>(
+      "RelayerFeeRouter",
+      [originXappConnectionManager.address],
+      upgradeBeaconController.address,
+    );
```

Comment:
we dont have to use the same proxy layout as nomad -- we can use the ones that exist in the current `Connext.sol` so we can make use of the existing tools.

Reply:
Updated the tests to use a transparent proxy for ConnextHandler and RelayerFeeRouter https://github.com/connext/nxtp/pull/1026/commits/c5361bfc30b6f4541676f8d3988be56dcd10c58e